webapp/channels/src/components/admin_console/billing/delete_workspace/progress_modal.tsx

```diff
@@ -2,8 +2,7 @@
 // See LICENSE.txt for license information.
```

Comment:
looks like I missed this one in the self serve clean up - if you want to remove it now you can (looks to not be imported) otherwise you can leave it for me to get later

Reply:
I think I'll leave those for you in case there's any related components or anything else to remove. I'm trying my best to keep these changes to just i18n-related code since I've already spent too much time on them